### DISCUSSION TITLE
stream download multiple csv files at once.

### DISCUSSION THREAD

**Discussion by abdulrehmandar2234** [spatie/simple-excel#108] [Q&A]
Hi, I am trying to create 2 CSV file and download it to the browser. but once the first file is streamed the code doesn't go to the next line. I have more lines after these two streams. only the first steam is working. Anything I am doing wrong. can anyone help?

```
   SimpleExcelWriter::streamDownload('updatedContacts.csv') //exporting updated contacts
    ->addRows($transactions);

    SimpleExcelWriter::streamDownload('deletedContacts.csv') //exporting deleted contacts
    ->addRows($deletedRecords);

**Comment by shaedrich** [comment]
That's the related error I'm getting:
> Cannot modify header information - headers already sent by (output started at [&hellip;]\vendor\openspout\openspout\src\Writer\Common\Helper\ZipHelper.php:197) {"exception":"[object] (ErrorException(code: 0): Cannot modify header information - headers already sent by (output started at [&hellip;]\\vendor\\openspout\\openspout\\src\\Writer\\Common\\Helper\\ZipHelper.php:197) at [&hellip;]\\vendor\\openspout\\openspout\\src\\Writer\\AbstractWriter.php:87)
